app/assets/src/components/views/visualizations/VisualizationsView.jsx

```diff
@@ -38,9 +50,9 @@ class VisualizationsView extends React.Component {
     return (
       <div>
         {publicAccess ? (
-          <GlobeIcon className={cx(cs.icon, cs.iconGlobe)} />
+          <PublicProjectIcon className={cx(cs.icon, cs.iconPublic)} />
```

Comment:
@happyimadesignr we need two new icons here for visualizations...

Reply:
is there some way to extract them from https://chanzuckerberg.invisionapp.com/d/main#/console/16827472/350949082/preview ?

Reply:
There could be, but Jenn has to explicitly make it available as an asset (which she did not do yet). 
Just ask her directly, and she will send you the svg file.